spring-cloud-gcp-data-spanner/src/main/java/org/springframework/cloud/gcp/data/spanner/repository/query/SqlSpannerQuery.java

```diff
@@ -301,11 +301,10 @@ else if (expression instanceof CompositeStringExpression) {
 
 		String sql;
 
-		QueryTagValue(List<String> tags, Object[] rawParams, Object[] params, String sql,
-				SpannerMappingContext spannerMappingContext) {
+		QueryTagValue(List<String> tags, Object[] rawParams, Object[] params, String sql) {
 			this.tags = tags;
 			this.intialParams = params;
-			this.sql = resolveEntityClassNames(sql, spannerMappingContext);
+			this.sql = sql;
```

Comment:
I knew that, but you requested to modify the constructor :)

Reply:
not that constructor :)